src/sagas/helpers.js

```diff
@@ -48,3 +48,21 @@ export function* dispatchAndWait(action, successAction, failureAction) {
     falure: take(failureAction),
   });
 }
+
+/**
+ * Handles errors thrown from the main saga (started with call) by yielding
+ * an action passed as a parameter
+ *
+ * @param action - The action to call
```

Comment:
Is it missing the `saga` parameter?

Reply:
No, but the parameter was wrong, fixed in 82c136916b8e9e120a079bfa83a532815dc05c4d